dist/select.js

```diff
@@ -2741,7 +2741,7 @@
               for (var t, i = Object(this), n = i.length >>> 0, s = arguments[1], o = 0; o < n; o++) {
                 if (t = i[o], e.call(s, t, o, i)) return t;
               }
-            }), e.prototype = window.Event.prototype, window.CustomEvent = e;
+            });
```

Comment:
Code is van Choices.js en dient om IE11 bug op te lossen. Wij voorzien (standaard) geen IE11 ondersteuning, dus op zich heb ik geen problemen met de wijziging, maar het vergt wel een extra wijziging bij een toekomstige update. Was deze wijziging nodig? Voor niet IE11 browsers zou ik geen verschil verwachten?

Reply:
Het zit ver maar het had iets te maken met bubbles en capture properties van het event. Dit zorgde ervoor dat we die properties niet konden zetten als we die zelf uitzonden. Iets in die aard

Reply:
Anders moet je eens testen na de merge van PR #50 of het probleem zich nog steeds stelt. Afhankelijk van de feedback zullen we zien hoe we deze wijziging verwerken en kunnen voorkomen dat het probleem zich in de toekomst opnieuw voordoet.

Reply:
Gelinkt aan https://github.com/milieuinfo/webcomponent-vl-ui-checkbox/pull/17.